boards/rpi-pico-w/Makefile

```diff
@@ -1,2 +1,2 @@
-DIRS = $(RIOTBOARD)/rpi-pico
+DIRS = $(RIOTBOARD)/rpi-pico-w
```

Comment:
`rpi-pico-w` inherits its configuration `rpi-pico` so I am wondering why you need this ?
Did you encounter any issue with `rpi-pico-w` ?

Reply:
Not that I remember, I think I just thought the pico-w was without a sufficient board definition